src/Components/WorkloadsListTable/WorkloadsListTable.js

```diff
@@ -44,32 +44,28 @@ const WorkloadsListTable = ({
   //const workloads = data?.workloads || [];
   //to check all types of filters use the mockdata json
   const workloads = mockdata;
+  const perPage = filters.limit;
+  const page = filters.offset / filters.limit + 1;
```

Comment:
I feel we better have rounding here. Imagine offset set to 50 but limit to 15. the result page is 4.333333333333334. This scenario shouldn't happen if all of our pagination constants are divisible, but it can cause bugs theoretically with slightly other variations in the future..